#goal
bool start
float64 goal_x
float64 goal_y
float64 goal_z
---
#result
bool finish
---
#feedback
bool isMoving